.travis.yml

```diff
@@ -66,7 +66,8 @@ script:
  - if [[ -z $CIBW_BEFORE_BUILD ]]; then py.test -v --timeout 20 --ignore=tmp-build --import-mode append ; fi
  - if [[ -n $TRAVIS_TAG && -n $CIBW_BEFORE_BUILD ]]; then cibuildwheel --output-dir wheelhouse1 && tools/fixup-wheels.sh wheelhouse1 wheelhouse ; fi
  - if [[ -n $TRAVIS_TAG && $TRAVIS_OS_NAME == linux && -n $CIBW_BEFORE_BUILD ]]; then tools/test-manylinux.sh ; fi
-
+ - if [[ -n $TRAVIS_TAG && -n $CIBW_BEFORE_BUILD ]]; then pip install --no-index -f wheelhouse/ confluent-kafka;\
+		 py.test -v --timeout 20 --ignore=tmp-build --import-mode append; tests/test_misc.py::test_unordered_dict; fi 
```

Comment:
`... append; tests/test_misc...`

Should there really be a ; in there?

Also, the tab indent is very red in the diff

Reply:
You should probably use `&&` instead of `;` between pip install and py.test.
Another thing to note is that it is error prone to run the tests from within the confluent-kafka-python source tree since `import confluent_kafka` may pick up the local directory, better to cd out to the parent directory.